seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/cdc/CdcRowTransformConfig.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.transform.cdc;
+
+import org.apache.seatunnel.api.configuration.Option;
+import org.apache.seatunnel.api.configuration.Options;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+public class CdcRowTransformConfig implements Serializable {
+
+    public static final String PLUGIN_NAME = "CdcRow";
```

Comment:
It's a weird and unintuitive name

Reply:
@Hisoka-X @hailin0  What name would be more appropriate to discuss this converter

Reply:
how about `RowKindExtractor`

Reply:
I am considering extending other CDC fields later, such as ts_ms, database_name, table_name, etc.

Reply:
> `ts_ms, database_name, table_name`

Not only cdc contains these fields, so I think use name without `CDC` should be better.